google/cloud/pubsublite/internal/stream_factory.h

```diff
@@ -0,0 +1,85 @@
+#include "absl/container/flat_hash_map.h"
+#include <google/cloud/pubsublite/v1/publisher.grpc.pb.h>
+
+namespace google {
+namespace cloud {
+namespace pubsublite_internal {
+
+template <class Request, class Response>
+using BidiStream = google::cloud::internal::AsyncStreamingReadWriteRpc<Request, Response>;
```

Comment:
I am missing some `#include` for this file?  Probably this file is not compiled, consider adding a test, which would at least force the compilation of this one file.

Reply:
This now runs the bazel CI on my machine.